modules/util/source-formatter/src/test/java/com/liferay/source/formatter/processor/UpgradeSourceProcessorTest.java

```diff
@@ -348,11 +348,17 @@ public void testUpgradeJavaUpdateFileEntryMethodCheck() throws Exception {
 	public void testUpgradeJavaUserLocalServiceUtilCheck() throws Exception {
 		test(
 			"upgrade/UpgradeJavaUserLocalServiceUtilCheck.testjava",
-			StringBundler.concat(
-				"Could not resolve types of updateStatus method. The method ",
-				"signature has changed to updateStatus(long userId,",
-				"int status, ServiceContext serviceContext). Fill the new ",
-				"parameter manually."));
+			new String[] {
+				StringBundler.concat(
+					"Could not resolve types of addUser method. The method ",
+					"signature has changed. Fill the new parameter manually, ",
+					"see LPS-192661 and LPS-196617"),
+				StringBundler.concat(
+					"Could not resolve types of updateStatus method. The ",
+					"method signature has changed to updateStatus(long userId,",
+					"int status, ServiceContext serviceContext). Fill the new ",
+					"parameter manually.")
```

Comment:
@TamyrisTorres, why only in this case do you describe the new method's signature, and why only in this case don't have the ticketId?

Reply:
In this case, I can be specific in the message, because it is just one method to change. 
But in the other case, the message must explain two cases to change.

Reply:
resolved